lstchain/tools/lstchain_create_calibration_file.py

```diff
@@ -238,8 +238,8 @@ def finish(self):
 
         self.log.info(f"Written {self.n_calib} calibration events")
         if self.n_calib == 0:
-             self.log.critical(f"!!! No calibration events in the output file !!! : ")
-             self.log.critical(f"flatfiels collected statistics = {self.processor.flatfield.num_events_seen} events")
+             self.log.critical("!!! No calibration events in the output file !!! : ")
+             self.log.critical(f"flatfield collected statistics = {self.processor.flatfield.num_events_seen} events")
```

Comment:
It doesn't really matter here, since critical messages will probably always be printed, but for `info` and `debug` messages there is a big difference:

For logging messages, you shouldn't use f-strings, but the formatting supported by the logging messages itself, since these are "lazy": if the message is not printed, the potentially expensive formatting is not done:

```python
self.log.critical("Flatfield collected statistics = %d", self.processor.flatfield.num_events_seen")
```
```

Reply:
I see, goof to know. I will take this into account in the future. Since these messages are pretty short, if you agree, I prefer to go ahead with other developments.